tony-azkaban/src/test/java/com/linkedin/tony/azkaban/TestTonyJob.java

```diff
@@ -65,6 +72,10 @@ public String getWorkingDirectory() {
     };
     String args = tonyJob.getMainArguments();
     Assert.assertTrue(args.contains(TonyJobArg.HDFS_CLASSPATH.tonyParamName + " hdfs://nn:8020"));
+    Assert.assertTrue(args.contains(TonyJobArg.SHELL_ENV.tonyParamName + " " + azkabanInputDataset
+        + "=HDFS_DYNAMIC_PATH:input_ds:/jobs/azktest/gsalia_tony/192/input_ds_job1"));
```

Comment:
there should be single quotes around the path here?

Reply:
Fixed.